.travis.yml

```diff
@@ -37,7 +37,7 @@ install:
   - pip install --upgrade git+git://github.com/cpcloud/redshift_sqlalchemy
 
   # install pyspark
-  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install -c blaze spark; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install -c blaze -c https://conda.binstar.org/blaze/channel/dev spark; fi
```

Comment:
For now in dev while I do some house keeping in conda-cluster
On Mar 8, 2015 5:16 PM, "Matthew Rocklin" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/ContinuumIO/odo/pull/117#discussion_r26009851>:
>
> > @@ -37,7 +37,7 @@ install:
> >    - pip install --upgrade git+git://github.com/cpcloud/redshift_sqlalchemy
> >
> >    # install pyspark
> > -  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install -c blaze spark; fi
> > +  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install -c blaze -c https://conda.binstar.org/blaze/channel/dev spark; fi
>
> Should that be in main cc @quasiben <https://github.com/quasiben>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/odo/pull/117/files#r26009851>.
>